examples/pytorch/dimenet/README.md

```diff
@@ -0,0 +1,105 @@
+# DGL Implementation of DimeNet
+
+This DGL example implements the GNN model proposed in the paper [Directional Message Passing for Molecular Graphs](https://arxiv.org/abs/2003.03123) and [Fast and Uncertainty-Aware Directional Message Passing for Non-Equilibrium Molecules](https://arxiv.org/abs/2011.14115). For the original implementation, see [here](https://github.com/klicperajo/dimenet).
+
+Contributor: [xnuohz](https://github.com/xnuohz)
+
+### Requirements
+The codebase is implemented in Python 3.6. For version requirement of packages, see below.
+
+```
+click 7.1.2
+dgl 0.5.3
+logzero 1.6.3
+numpy 1.19.5
+pandas 1.1.5
+ruamel.yaml 0.16.12
+scikit-learn 0.24.1
+scipy 1.5.4
+sympy 1.7.1
+torch 1.7.0
+tqdm 4.56.0
+```
+
+### The graph datasets used in this example
+
+The DGL's built-in QM9 dataset. Dataset summary:
+
+* Molecular Graphs: 13,0831
+* Number of Tasks: 12
+
+### Usage
+
+**Note: DimeNet++ is recommended to use instead of DimeNet.**
```

Comment:
Is it possible to summarize or briefly explain the differences between DimeNet++ and DimeNet? Alternatively, what are the advantages of DimeNet++ over DimeNet?

Reply:
1. The bilinear layer in DimeNet is replaced by a simple Hadamard product;
2. Less interaction blocks.